### DISCUSSION TITLE
官方公告或许可以加个推送类别的开关吗

### DISCUSSION THREAD

**Discussion by Xm798** [Arondight/Adachi-BOT#771] [General]
活动 公告 资讯这三个类别能选择性关闭嘛
![image](https://user-images.githubusercontent.com/48074861/161043064-f62aaafd-95d8-466d-829d-45dd328320e4.png)

**Comment by Arondight** [comment]
拉一下代码试试

**Reply by Xm798** [reply]
好耶！好快！
